webpack/base.config.js

```diff
@@ -0,0 +1,39 @@
+import webpack from 'webpack';
+import yargs from 'yargs';
+
+export const argv = yargs
+  .alias('p', 'optimize-minimize')
+  .alias('d', 'debug')
+  .option('port', {
+    default: '8080',
+    type: 'string'
+  })
+  .argv;
+
+const baseConfig = {
+  entry: undefined,
+
+  output: undefined,
+
+  externals: undefined,
+
+  module: {
+    loaders: [
+      { test: /\.js/, loader: 'babel', exclude: /node_modules/ }
+    ]
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify(argv.optimizeMinimize ? 'production' : 'development')
+      }
+    })
+  ]
+};
+
+if (argv.optimizeMinimize) {
+  baseConfig.devtool = 'source-map';
+}
```

Comment:
This came up before, but I also like `eval-source-map` - lets you see the original ES6 code, and is a lot faster when running in "watch" mode than normal source maps. I think we saw that it didn't properly map traceback lines in some cases, though.